lib/version.js

```diff
@@ -0,0 +1,108 @@
+var /**
+     * @const
+     * @type {String[]}
+     */
+    PACKAGES_TO_CONSIDER = [
+        'chai-postman',
+        'postman-collection',
+        'postman-request',
+        'postman-sandbox',
+        'uniscope',
+        'uvm'
+    ],
+
+    /**
+     * Generate dependency tree of runtime.
```

Comment:
Need to update the description to indicate it returns dependencies versions as well.

Reply:
How does it look now?